packages/endomoat/src/index.js

```diff
@@ -0,0 +1,145 @@
+const ENDO_ROOT_POLICY = 'root'
+const ENDO_WRITE_POLICY = 'write'
+const ENDO_WILDCARD_POLICY = 'any'
+const LAVAMOAT_ROOT_PKG = '<root>'
+const LAVAMOAT_PKG_POLICY_ROOT = '$root$'
+
+const RSRC_POLICY_PKGS = 'packages'
+const RSRC_POLICY_BUILTINS = 'builtins'
+const RSRC_POLICY_GLOBALS = 'globals'
+
+const {isArray} = Array
+
+/**
+ * @typedef {typeof ENDO_ROOT_POLICY} RootPolicy
+ * @typedef {typeof ENDO_WRITE_POLICY} WritePolicy
+ */
+
+/**
+ * @typedef {import('@endo/compartment-mapper').Policy<RootPolicy>} LavaMoatEndoPolicy
+ */
+
+const DEFAULT_ATTENUATOR = '@lavamoat/endomoat/attenuator/default'
+const PROPERTY_ATTENUATOR = '@lavamoat/endomoat/attenuator/property'
+
+/**
+ * @param {Record<string, boolean>} item - A value in `ResourcePolicy`
+ */
+function toEndoRsrcPkgsPolicyBuiltins(item) {
+  /** @type {Exclude<import('@endo/compartment-mapper').PackagePolicy['builtins'], undefined>} */
+  const policyItem = {}
+  for (const [key, value] of Object.entries(item)) {
+    if (key.includes('.')) {
+      let [builtinName, ...rest] = key.split('.')
+      let propName = rest.join('.')
+      const itemForBuiltin = policyItem[builtinName]
+      if (typeof itemForBuiltin === 'boolean') {
+        throw new TypeError('Expected a FullAttenuationDefinition; got a boolean')
+      }
+      if (isArray(itemForBuiltin)) {
+        throw new TypeError('Expected a FullAttenuationDefinition; got an array')
+      }
+      const otherParams = itemForBuiltin?.params ?? []
+      policyItem[builtinName] = {
+        attenuate: PROPERTY_ATTENUATOR,
+        params: [...otherParams, propName],
+      }
+    } else {
+      policyItem[key] = value
+    }
+  }
+  return policyItem
+}
+
+/**
+ * @param {Record<string, boolean>} item - A value in `ResourcePolicy`
+ */
+function toEndoRsrcPkgsPolicyPkgs(item) {
+  /** @type {Exclude<import('@endo/compartment-mapper').PackagePolicy['packages'], undefined>} */
+  const policyItem = {}
+  for (const [key, value] of Object.entries(item)) {
+    if (key === LAVAMOAT_PKG_POLICY_ROOT) {
+      // unsure what to do here
```

Comment:
lavamoat policy should not specify anything about the root package as it's allowed all by default. We need to generate the item in endo policy that defines our expected behavior for root.

We'll likely need an attenuator there when we get to supporting `"write"` policy values.